app/assets/javascripts/dance_comments/new.js

```diff
@@ -17,35 +17,33 @@ class NewComment{
         <footer>
           <ul>
             <li><a href="/dance_comments/${this.comment[0]}/edit">Edit</a></li>
-            <li id="remove_item"><a href="">Delete</a></li>
+            <li><a href="">Delete</a></li>
           </ul>
         </footer>
       </article>
     </li>`;
   }
 }
 
-let prependTarget = (target, text) => {
-  target.prepend(text);
-};
-
-submitClick = (event) => {
+submitDCClick = (event) => {
   event.preventDefault();
 
-  let tId = $('#dance_comment_comment_type_id').val();
-  let com = $('#dance_comment_comment').val();
-  let uId = $('#user-id').text();
-  let dId = $('#dance-id').text();
+  let comment = {
+    comment: $('#dance_comment_comment').val(),
+    comment_type_id: $('#dance_comment_comment_type_id').val(),
+    user_id: $('#user-id').text(),
+    dance_id: $('#dance-id').text()
+  };
```

Comment:
I think I understand. But can the class pull the information from the doc or does it need to be passed to the constructor?

```class NewComment{
  constructor() {
    this.comment = $('#dance_comment_comment').val();
    this.comment_type_id = $('#dance_comment_comment_type_id').val();
    this.user_id = $('#user-id').text();
    this.dance_id = $('#dance-id').text();
  }
}

submitDCClick = (event) => {
  let newCom = new NewComment();
}
```
OR
```class NewComment{
  constructor(arr) {
    this.comment = arr.comment;
    this.comment_type_id = arr.comment_type_id;
    this.user_id = arr.user.id;
    this.dance_id = arr.dance_id;
  }
}

submitDCClick = (event) => {
  let comment = {
    comment: $('#dance_comment_comment').val(),
    comment_type_id: $('#dance_comment_comment_type_id').val(),
    user_id: $('#user-id').text(),
    dance_id: $('#dance-id').text()
  }; 
  let newCom = new NewComment(comment);
}
```